src/server/listener.cc

```diff
@@ -220,7 +222,7 @@ Listener::run() {
                 else {
                     if (event.flags.hasFlag(Polling::NotifyOn::Read)) {
                         auto fd = event.tag.value();
-                        if (fd == listen_fd)
+                        if ((ssize_t)fd == listen_fd)
```

Comment:
Listener::listen_fd is an int (as it should be for file descriptors).  Just replace lines 224,255 with:

if (static_cast<int>(event.tag.value()) == listen_fd)

Unless "value()" already returns an int, then omit the static_cast.

Reply:
`Tag::value()` returns a `uint64_t`, so the cast is necessary.